src/main/java/Dao/CartItemDao.java

```diff
@@ -0,0 +1,76 @@
+package Dao;
+
+import Entities.CartItem;
+

```

Comment:
Make sure all your classes are properly formatted according to the java code convention.